packages/smooth_app/lib/services/logs/logs_fimber_impl.dart

```diff
@@ -0,0 +1,118 @@
+import 'dart:io';
+
+import 'package:fimber/fimber.dart';
+import 'package:flutter/foundation.dart';
+import 'package:path/path.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:smooth_app/services/logs/fimber/fimber_helper.dart';
+import 'package:smooth_app/services/logs/fimber/trees/debug_fimber_tree.dart';
+import 'package:smooth_app/services/logs/fimber/trees/file_fimber_tree.dart';
+import 'package:smooth_app/services/logs/fimber/trees/sentry_fimber_tree.dart';
+import 'package:smooth_app/services/logs/smooth_log_levels.dart';
+import 'package:smooth_app/services/logs/smooth_logs_service.dart';
+
+/// On debug builds: device logs + file (all levels)
+/// On release builds : device logs (only errors), sentry (all levels) and
+/// file (only errors & info)
+class FimberLogImpl implements AppLogService {
+  // Link to a file tree impl (required for exporting logs)
+  late FileFimberTree _fileTree;
+
+  @override
+  Future<void> init() async {
+    if (kReleaseMode) {
+      Fimber.plantTree(
+        DebugTree(
+          logLevels: <String>['E'],
+        ),
+      );
+      _fileTree = FileFimberTree(
+        outputFile: await _fileName,
+        logLevels: LogLevels.prodLogLevels,
+      );
+      Fimber.plantTree(
+        SentryFimberTree(logLevels: LogLevels.allLogLevels),
```

Comment:
Shouldn't this also be just errors

Reply:
It's like a remote debug mode, that's why I prefer sending everything here